From c73d736c5d9fb71c53682c20f81160dde223ba21 Mon Sep 17 00:00:00 2001
From: Try Omarchy Release <actions@users.noreply.github.com>
Date: Wed, 21 Oct 2026 22:28:04 +0000
Subject: [PATCH] Refresh the guest package lock

avahi 1:0.9rc5-1 -> 1:0.9rc5-2
bubblewrap 0.13.0-1 -> 0.13.1-1
ca-certificates-mozilla 3.131-1 -> 3.131.1-1
chromium 154.0.8037.97-1 -> 155.0.8059.84-1
clang 23.1.1-1 -> 23.1.3-4
compiler-rt 23.1.1-1 -> 23.1.3-4
curl 8.22.0-1 -> 8.23.0-1
dbus 1.16.2-1 -> 1.16.2-2
dua-cli 2.45.1-1 -> 2.46.0-1
elfutils 0.196-1 -> 0.197-1
fastfetch 2.69.0-1 -> 2.70.0-1
freetype2 2.14.3-1 -> 2.15.0-1
glycin 2.2.2-1 -> 2.2.3-1
glycin-gtk4 2.2.2-1 -> 2.2.3-1
gperftools 2.18.1-1 -> 2.19-1
graphicsmagick 1.3.49-2 -> 1.3.49-3
graphviz 16.1.0-1 -> 16.1.0-2
hyprland-preview-share-picker 0.2.1-1 -> 0.2.1-3
imagemagick 7.1.2.33-3 -> 7.1.2.34-2
iproute2 7.2.0-1 -> 7.3.0-1
libarchive 3.8.9-1 -> 3.8.10-1
libbs2b 3.1.0-10 -> 3.1.0-11
libcolord 1.4.8-1 -> 1.4.8-2
libei 1.6.0-1 -> 1.6.0-2
libelf 0.196-1 -> 0.197-1
libgudev 238-3 -> 238-4
libmm-glib 1.24.2-1 -> 1.24.2-2
libnotify 0.8.8-1 -> 0.8.9-1
libpulse 17.0+r98+gb096704c0-1 -> 17.0+r134+g77d25a1e6-1
librsvg 2:2.63.2-1 -> 2:2.63.3-1
libx11 1.8.13-2 -> 1.8.14-1
libxi 1.8.3-1 -> 1.8.4-1
libxpm 3.5.19-1 -> 3.5.20-1
libxrender 0.9.12-1 -> 0.9.13-1
libxtst 1.2.5-1 -> 1.2.6-1
linux 7.2.9.arch1-1 -> 7.2.11.arch1-1
linux-api-headers 7.2-1 -> 7.3-1
linux-headers 7.2.9.arch1-1 -> 7.2.11.arch1-1
llvm-libs 23.1.1-1 -> 23.1.3-4
luajit 2.1.1791890110+634daf2-1 -> 2.1.1792588164+a4c0645-1
mesa 1:26.2.4-1 -> 1:26.2.5-1
noto-fonts-emoji 1:2.051-1 -> 1:2.057-1
nss 3.131-1 -> 3.131.1-1
omacalc 0.2.2-1 -> 0.3.1-1
omacut 0.4.0-1 -> 0.7.2-1
omarchy-nvim 2026.8.13-1 -> 2026.9.21-5
omawrite 0.5.0-1 -> 0.7.0-1
openssh 10.6p1-1 -> 10.6p1-2
pacman 7.1.0.r9.g54d9411-2 -> 7.1.0.r9.g54d9411-3
pcsclite 2.5.2-1 -> 2.5.3-1
perl 5.42.3-1 -> 5.44.0-1
pkgconf 3.0.7-1 -> 3.0.10-1
polkit 127-3 -> 128-1
poppler 26.08.0-1 -> 26.10.0-1
poppler-glib 26.08.0-1 -> 26.10.0-1
qt6-base 6.12.0-2 -> 6.12.0-3
qt6-declarative 6.12.0-1 -> 6.12.0-2
speex 1.2.1-2 -> 1.2.1-3
speexdsp 1.2.1-2 -> 1.2.1-3
starship 1.26.0-1 -> 1.27.0-1
systemd 262-1 -> 262.1-1
systemd-libs 262-1 -> 262.1-1
systemd-resolvconf 262-1 -> 262.1-1
systemd-sysvcompat 262-1 -> 262.1-1
texinfo 7.3-1 -> 7.3-3
uwsm 0.27.0-1 -> 0.27.1-1
vulkan-mesa-implicit-layers 1:26.2.4-1 -> 1:26.2.5-1
vulkan-swrast 1:26.2.4-1 -> 1:26.2.5-1
vulkan-virtio 1:26.2.4-1 -> 1:26.2.5-1
webrtc-audio-processing-1 1.3-5 -> 1.3-6
xdg-desktop-portal-gtk 1.15.3-1 -> 1.15.3-2
xorg-server 21.1.25-1 -> 21.1.25-2
xorg-server-common 21.1.25-1 -> 21.1.25-2
clang-libs 23.1.3-4 (new)
llvm 23.1.3-4 (new)
llvm-lit 23.1.3-4 (new)
simdutf 9.2.1-2 (new)
---
 guest/packages.lock.json | 150 ++++++++++++++++++++-------------------
 1 file changed, 77 insertions(+), 73 deletions(-)

diff --git a/guest/packages.lock.json b/guest/packages.lock.json
index a1a0750..cf0e844 100644
--- a/guest/packages.lock.json
+++ b/guest/packages.lock.json
@@ -19,7 +19,7 @@
     "audit": "4.2.1-1",
     "autoconf": "2.73-1",
     "automake": "1.19-1",
-    "avahi": "1:0.9rc5-1",
+    "avahi": "1:0.9rc5-2",
     "base": "3-3",
     "base-devel": "1-2",
     "bash": "5.3.20-1",
@@ -31,25 +31,26 @@
     "boost-libs": "1.92.0-1",
     "brotli": "1.2.0-1",
     "btop": "1.4.7-1",
-    "bubblewrap": "0.13.0-1",
+    "bubblewrap": "0.13.1-1",
     "bzip2": "1.0.8-6",
     "ca-certificates": "20240618-1",
-    "ca-certificates-mozilla": "3.131-1",
+    "ca-certificates-mozilla": "3.131.1-1",
     "ca-certificates-utils": "20240618-1",
     "cairo": "1.18.6-1",
-    "chromium": "154.0.8037.97-1",
-    "clang": "23.1.1-1",
+    "chromium": "155.0.8059.84-1",
+    "clang": "23.1.3-4",
+    "clang-libs": "23.1.3-4",
     "cmake": "4.4.4-1",
-    "compiler-rt": "23.1.1-1",
+    "compiler-rt": "23.1.3-4",
     "coreutils": "9.12-2",
     "cpio": "2.15-3",
     "cppdap": "1.58.0-3",
     "cpptrace": "1.0.4-2",
     "cryptsetup": "2.8.8-1",
-    "curl": "8.22.0-1",
+    "curl": "8.23.0-1",
     "dav1d": "1.5.4-1",
     "db5.3": "5.3.28-7",
-    "dbus": "1.16.2-1",
+    "dbus": "1.16.2-2",
     "dbus-broker": "37-3",
     "dbus-broker-units": "37-3",
     "dbus-units": "37-3",
@@ -61,17 +62,17 @@
     "diffutils": "3.12-2",
     "dkms": "3.4.4-1",
     "double-conversion": "3.4.0-1",
-    "dua-cli": "2.45.1-1",
+    "dua-cli": "2.46.0-1",
     "duktape": "2.7.0-7",
     "e2fsprogs": "1.47.4-1",
-    "elfutils": "0.196-1",
+    "elfutils": "0.197-1",
     "enchant": "2.8.21-1",
     "exempi": "2.6.6-3",
     "exiv2": "0.28.9-1",
     "expat": "2.9.0-1",
     "eza": "0.23.5-2",
     "fakeroot": "1:1.37.2-3",
-    "fastfetch": "2.69.0-1",
+    "fastfetch": "2.70.0-1",
     "fcft": "3.3.3-1",
     "fcitx5": "5.1.23-1",
     "fcitx5-chewing": "5.1.13-1",
@@ -87,7 +88,7 @@
     "flex": "2.6.4-6",
     "fontconfig": "2:2.18.3-2",
     "foot": "1.28.0-2",
-    "freetype2": "2.14.3-1",
+    "freetype2": "2.15.0-1",
     "fribidi": "1.0.17-1",
     "fuse-common": "3.18.3-1",
     "fuse3": "3.18.3-1",
@@ -114,8 +115,8 @@
     "glibc": "2.44+r50+g1848099f063e-1",
     "glslang": "1:1.4.363.0-1",
     "glu": "9.0.3-3",
-    "glycin": "2.2.2-1",
-    "glycin-gtk4": "2.2.2-1",
+    "glycin": "2.2.3-1",
+    "glycin-gtk4": "2.2.3-1",
     "gmp": "6.3.0-3",
     "gnome-autoar": "0.5.2-1",
     "gnome-desktop-4": "1:51.0-1",
@@ -125,15 +126,15 @@
     "gnupg": "2.4.9-3",
     "gnutls": "3.8.13-3",
     "gobject-introspection-runtime": "1.86.0-2",
-    "gperftools": "2.18.1-1",
+    "gperftools": "2.19-1",
     "gpgme": "2.2.0-1",
     "gpgmepp": "2.2.0-1",
     "gpm": "1.20.7.r38.ge82d1a6-7",
     "gpu-screen-recorder": "6.1.4-1",
     "graphene": "1.10.8-2",
-    "graphicsmagick": "1.3.49-2",
+    "graphicsmagick": "1.3.49-3",
     "graphite": "1:1.3.15-1",
-    "graphviz": "16.1.0-1",
+    "graphviz": "16.1.0-2",
     "grep": "3.12-2",
     "grim": "1.5.0-2",
     "groff": "1.24.2-1",
@@ -169,7 +170,7 @@
     "hypridle": "0.1.8-2",
     "hyprland": "0.56.2-4",
     "hyprland-guiutils": "0.2.2-4",
-    "hyprland-preview-share-picker": "0.2.1-1",
+    "hyprland-preview-share-picker": "0.2.1-3",
     "hyprlang": "0.6.8-5",
     "hyprpicker": "0.4.7-4",
     "hyprsunset": "0.4.0-3",
@@ -180,11 +181,11 @@
     "iana-etc": "20260617-1",
     "icu": "78.3-1",
     "ijs": "0.35-7",
-    "imagemagick": "7.1.2.33-3",
+    "imagemagick": "7.1.2.34-2",
     "imv": "5.0.1-2",
     "iniparser": "4.3.2-1",
     "inotify-tools": "4.26.270-1",
-    "iproute2": "7.2.0-1",
+    "iproute2": "7.3.0-1",
     "iptables": "1:1.8.13-1",
     "iputils": "20250605-1",
     "iso-codes": "4.20.1-1",
@@ -208,7 +209,7 @@
     "leptonica": "1.87.0-2",
     "less": "1:710-1",
     "libadwaita": "1:1.10.0-1",
-    "libarchive": "3.8.9-1",
+    "libarchive": "3.8.10-1",
     "libasan": "16.2.1+r23+gd564253eb6c8-1",
     "libass": "0.17.5-1",
     "libassuan": "3.0.0-1",
@@ -229,7 +230,7 @@
     "libblockdev-swap": "3.5.0-2",
     "libbluray": "1.5.1-1",
     "libbpf": "1.7.0-1",
-    "libbs2b": "3.1.0-10",
+    "libbs2b": "3.1.0-11",
     "libbsd": "0.12.2-2",
     "libbytesize": "2.12-3",
     "libcap": "2.78-1",
@@ -238,7 +239,7 @@
     "libcdio-paranoia": "10.2+2.0.2-3",
     "libchewing": "0.13.1-1",
     "libcloudproviders": "0.4.1-1",
-    "libcolord": "1.4.8-1",
+    "libcolord": "1.4.8-2",
     "libcue": "2.3.0-2",
     "libcups": "2:2.4.20-1",
     "libdaemon": "0.14-6",
@@ -254,8 +255,8 @@
     "libdwarf": "1:2.3.3-1",
     "libebur128": "1.2.6-2",
     "libedit": "20260512_3.1-1",
-    "libei": "1.6.0-1",
-    "libelf": "0.196-1",
+    "libei": "1.6.0-2",
+    "libelf": "0.197-1",
     "libepoxy": "1.5.10-3",
     "libevdev": "1.14.0-1",
     "libevent": "2.1.13-2",
@@ -273,7 +274,7 @@
     "libgomp": "16.2.1+r23+gd564253eb6c8-1",
     "libgpg-error": "1.61-1",
     "libgsf": "1.14.60-1",
-    "libgudev": "238-3",
+    "libgudev": "238-4",
     "libgxps": "0.3.2-5",
     "libheif": "1.23.6-1",
     "libhwasan": "16.2.1+r23+gd564253eb6c8-1",
@@ -299,7 +300,7 @@
     "libmakepkg-dropins": "20-2",
     "libmd": "1.3.0-1",
     "libmediainfo": "26.10-1",
-    "libmm-glib": "1.24.2-1",
+    "libmm-glib": "1.24.2-2",
     "libmms": "0.6.4-6",
     "libmnl": "1.0.5-2",
     "libmodplug": "0.8.9.0-7",
@@ -317,7 +318,7 @@
     "libnice": "0.1.24-1",
     "libnl": "3.12.0-1",
     "libnm": "1.58.1-2",
-    "libnotify": "0.8.8-1",
+    "libnotify": "0.8.9-1",
     "libnsbmp": "0.1.7-2",
     "libnsgif": "1.0.0-2",
     "libnsl": "2.0.1-2",
@@ -340,11 +341,11 @@
     "libportal-gtk4": "0.11.0-1",
     "libproxy": "0.5.12-1",
     "libpsl": "0.21.5-2",
-    "libpulse": "17.0+r98+gb096704c0-1",
+    "libpulse": "17.0+r134+g77d25a1e6-1",
     "libquadmath": "16.2.1+r23+gd564253eb6c8-1",
     "libraqm": "0.11.0-1",
     "libraw1394": "2.1.2-4",
-    "librsvg": "2:2.63.2-1",
+    "librsvg": "2:2.63.3-1",
     "libsamplerate": "0.2.2-3",
     "libsasl": "2.1.28-5",
     "libseccomp": "2.6.0-1",
@@ -389,7 +390,7 @@
     "libwacom": "2.20.0-1",
     "libwebp": "1.6.0-2",
     "libwireplumber": "0.5.18-1",
-    "libx11": "1.8.13-2",
+    "libx11": "1.8.14-1",
     "libxau": "1.0.12-1",
     "libxcb": "1.17.0-1",
     "libxcomposite": "0.4.7-1",
@@ -402,22 +403,22 @@
     "libxfixes": "6.0.2-1",
     "libxfont2": "2.0.9-1",
     "libxft": "2.3.9-1",
-    "libxi": "1.8.3-1",
+    "libxi": "1.8.4-1",
     "libxinerama": "1.1.6-1",
     "libxkbcommon": "1.13.2-1",
     "libxkbcommon-x11": "1.13.2-1",
     "libxkbfile": "1.2.0-1",
     "libxml2": "2.15.4-1",
     "libxmu": "1.3.1-1",
-    "libxpm": "3.5.19-1",
+    "libxpm": "3.5.20-1",
     "libxpresent": "1.0.2-1",
     "libxrandr": "1.5.5-1",
-    "libxrender": "0.9.12-1",
+    "libxrender": "0.9.13-1",
     "libxshmfence": "1.3.3-1",
     "libxslt": "1.1.45-2",
     "libxss": "1.2.5-1",
     "libxt": "1.3.1-1",
-    "libxtst": "1.2.5-1",
+    "libxtst": "1.2.6-1",
     "libxv": "1.0.13-1",
     "libxxf86vm": "1.1.7-1",
     "libyaml": "0.2.5-3",
@@ -426,18 +427,20 @@
     "libzip": "1.11.4-1",
     "licenses": "20240728-1",
     "lilv": "0.28.0-1",
-    "linux": "7.2.9.arch1-1",
-    "linux-api-headers": "7.2-1",
-    "linux-headers": "7.2.9.arch1-1",
+    "linux": "7.2.11.arch1-1",
+    "linux-api-headers": "7.3-1",
+    "linux-headers": "7.2.11.arch1-1",
     "llhttp": "9.3.1-1",
-    "llvm-libs": "23.1.1-1",
+    "llvm": "23.1.3-4",
+    "llvm-libs": "23.1.3-4",
+    "llvm-lit": "23.1.3-4",
     "lm_sensors": "1:3.6.2-1",
     "lmdb": "0.9.35-1",
     "localsearch": "3.12.0-1",
     "lua": "5.5.1-1",
     "lua51-lpeg": "1.1.0-5",
     "lua54": "5.4.9-1",
-    "luajit": "2.1.1791890110+634daf2-1",
+    "luajit": "2.1.1792588164+a4c0645-1",
     "lv2": "1.18.10-2",
     "lz4": "1:1.10.0-2",
     "lzo": "2.10-5",
@@ -446,7 +449,7 @@
     "man-db": "2.13.1-2",
     "md4c": "0.6.0-2",
     "mdadm": "4.6-2",
-    "mesa": "1:26.2.4-1",
+    "mesa": "1:26.2.5-1",
     "ministream": "0.99.1-1",
     "minizip": "1:1.3.2-4",
     "mkinitcpio": "42.2-1",
@@ -469,29 +472,29 @@
     "nftables": "1:1.1.7-3",
     "noto-fonts": "1:2026.10.01-1",
     "noto-fonts-cjk": "20240730-1",
-    "noto-fonts-emoji": "1:2.051-1",
+    "noto-fonts-emoji": "1:2.057-1",
     "npth": "1.8-1",
     "nspr": "4.40-1",
-    "nss": "3.131-1",
+    "nss": "3.131.1-1",
     "numactl": "2.0.19-1",
     "ocl-icd": "2.3.5-1",
-    "omacalc": "0.2.2-1",
-    "omacut": "0.4.0-1",
+    "omacalc": "0.3.1-1",
+    "omacut": "0.7.2-1",
     "omarchy-keyring": "20251027-1",
-    "omarchy-nvim": "2026.8.13-1",
-    "omawrite": "0.5.0-1",
+    "omarchy-nvim": "2026.9.21-5",
+    "omawrite": "0.7.0-1",
     "oniguruma": "6.9.10-3",
     "openal": "1.25.2-2",
     "opencore-amr": "0.1.6-2",
     "openh264": "2.6.0-2",
     "openjpeg2": "2.5.4-1",
-    "openssh": "10.6p1-1",
+    "openssh": "10.6p1-2",
     "openssl": "3.6.5-1",
     "opus": "1.6.1-1",
     "orc": "0.4.44-1",
     "osinfo-db": "20260812-1",
     "p11-kit": "0.26.5-1",
-    "pacman": "7.1.0.r9.g54d9411-2",
+    "pacman": "7.1.0.r9.g54d9411-3",
     "pacman-contrib": "1.13.1-1",
     "pacman-mirrorlist": "20260610-1",
     "pahole": "1:1.32-1",
@@ -504,8 +507,8 @@
     "pciutils": "3.15.0-1",
     "pcre": "8.45-5",
     "pcre2": "10.49-1",
-    "pcsclite": "2.5.2-1",
-    "perl": "5.42.3-1",
+    "pcsclite": "2.5.3-1",
+    "perl": "5.44.0-1",
     "perl-error": "0.17030-3",
     "perl-mailtools": "2.22-3",
     "perl-timedate": "2.35-1",
@@ -515,13 +518,13 @@
     "pipewire-audio": "1:1.6.9-1",
     "pipewire-pulse": "1:1.6.9-1",
     "pixman": "0.46.4-1",
-    "pkgconf": "3.0.7-1",
+    "pkgconf": "3.0.10-1",
     "playerctl": "2.4.1-5",
     "plocate": "1.1.25-1",
-    "polkit": "127-3",
-    "poppler": "26.08.0-1",
+    "polkit": "128-1",
+    "poppler": "26.10.0-1",
     "poppler-data": "0.4.12-2",
-    "poppler-glib": "26.08.0-1",
+    "poppler-glib": "26.10.0-1",
     "popt": "1.19-2",
     "portaudio": "1:19.7.0-4",
     "procps-ng": "4.0.7-1",
@@ -535,8 +538,8 @@
     "python-yaml": "6.0.3-2",
     "qoi": "r350.4461cc3-1",
     "qrencode": "4.1.1-4",
-    "qt6-base": "6.12.0-2",
-    "qt6-declarative": "6.12.0-1",
+    "qt6-base": "6.12.0-3",
+    "qt6-declarative": "6.12.0-2",
     "qt6-multimedia": "6.12.0-1",
     "qt6-multimedia-ffmpeg": "6.12.0-1",
     "qt6-svg": "6.12.0-1",
@@ -560,6 +563,7 @@
     "shaderc": "2026.4-1",
     "shadow": "4.20.0.arch1-1",
     "shared-mime-info": "2.5.1-2",
+    "simdutf": "9.2.1-2",
     "slang": "2.3.3-4",
     "slurp": "1.5.0-2",
     "snappy": "1.3.1-1",
@@ -568,25 +572,25 @@
     "sord": "0.16.22-1",
     "source-highlight": "3.1.9-19",
     "spandsp": "0.0.6-7",
-    "speex": "1.2.1-2",
-    "speexdsp": "1.2.1-2",
+    "speex": "1.2.1-3",
+    "speexdsp": "1.2.1-3",
     "spirv-tools": "1:1.4.363.0-1",
     "sqlite": "3.54.0-1",
     "sratom": "0.6.22-1",
     "srt": "1.5.7-2",
-    "starship": "1.26.0-1",
+    "starship": "1.27.0-1",
     "sudo": "1.9.17.p2-6",
     "svt-av1": "4.2.0-1",
-    "systemd": "262-1",
-    "systemd-libs": "262-1",
-    "systemd-resolvconf": "262-1",
-    "systemd-sysvcompat": "262-1",
+    "systemd": "262.1-1",
+    "systemd-libs": "262.1-1",
+    "systemd-resolvconf": "262.1-1",
+    "systemd-sysvcompat": "262.1-1",
     "tar": "1.35-5",
     "tensaku": "0.29.0-2",
     "tesseract": "5.5.3-1",
     "tesseract-data-eng": "2:4.1.0-5",
     "tesseract-data-osd": "2:4.1.0-5",
-    "texinfo": "7.3-1",
+    "texinfo": "7.3-3",
     "tinysparql": "3.12.0-1",
     "tinyxml2": "11.0.0-2",
     "tmux": "3.8-1",
@@ -613,7 +617,7 @@
     "upower": "1.91.5-1",
     "util-linux": "2.42.4-1",
     "util-linux-libs": "2.42.4-1",
-    "uwsm": "0.27.0-1",
+    "uwsm": "0.27.1-1",
     "v4l-utils": "1.32.0-2",
     "v4l2loopback-dkms": "0.15.4-2",
     "vapoursynth": "80-1",
@@ -621,12 +625,12 @@
     "vmaf": "3.2.1-1",
     "volume_key": "0.3.12-12",
     "vulkan-icd-loader": "1.4.363.0-1",
-    "vulkan-mesa-implicit-layers": "1:26.2.4-1",
-    "vulkan-swrast": "1:26.2.4-1",
-    "vulkan-virtio": "1:26.2.4-1",
+    "vulkan-mesa-implicit-layers": "1:26.2.5-1",
+    "vulkan-swrast": "1:26.2.5-1",
+    "vulkan-virtio": "1:26.2.5-1",
     "wayland": "1.26.0-1",
     "wayland-protocols": "1.49-1",
-    "webrtc-audio-processing-1": "1.3-5",
+    "webrtc-audio-processing-1": "1.3-6",
     "which": "2.25-1",
     "wireplumber": "0.5.18-1",
     "wl-clipboard": "1:2.3.0-1",
@@ -645,7 +649,7 @@
     "xcb-util-renderutil": "0.3.10-2",
     "xcb-util-wm": "0.4.2-2",
     "xdg-desktop-portal": "1.22.1-2",
-    "xdg-desktop-portal-gtk": "1.15.3-1",
+    "xdg-desktop-portal-gtk": "1.15.3-2",
     "xdg-desktop-portal-hyprland": "1.4.1-2",
     "xdg-terminal-exec": "0.14.3-2",
     "xdg-user-dirs": "0.20-1",
@@ -654,8 +658,8 @@
     "xf86-input-libinput": "1.5.0-1",
     "xkeyboard-config": "2.48-1",
     "xorg-fonts-encodings": "1.1.0-2",
-    "xorg-server": "21.1.25-1",
-    "xorg-server-common": "21.1.25-1",
+    "xorg-server": "21.1.25-2",
+    "xorg-server-common": "21.1.25-2",
     "xorg-setxkbmap": "1.3.5-1",
     "xorg-xauth": "1.1.5-1",
     "xorg-xkbcomp": "1.5.0-1",
-- 
2.55.0

